hssqlppp/src/Database/HsSqlPpp/Parsing/ParserInternal.lhs

```diff
@@ -1206,10 +1206,15 @@ parse it
 >         pVol = matchAKeyword [("volatile", Volatile)
 >                              ,("stable", Stable)
 >                              ,("immutable", Immutable)]
->         readLang = keyword "language" *> matchAKeyword [("plpgsql", Plpgsql)
->                                                        ,("sql",Sql)]
+>         readLang = keyword "language" *> matchAKeyword [ ("plpgsql", Plpgsql)
```

Comment:
I'd prefer this using do notation and simpler indentation (break line and indent once before the list)